### DISCUSSION TITLE
在无图形界面的云服务器上创建模板项目，默认监听了localhost:5140，我该如何修改使我公网可以访问？

### DISCUSSION THREAD

**Discussion by k4m7v2pz** [koishijs/koishi#754] [Q&A] [answered]
node 16.16.0
yarn 1.22.19
![image](https://user-images.githubusercontent.com/97567575/179692918-2f5f192f-87fe-41fe-a4bf-774e52588444.png)

**Comment by MaikoTan** [comment, marked as answer]
Find the configuration file named `koishi.yml`, and append the line as the following:
```yaml
host: 0.0.0.0 # or you can change this to match your server IP address
```
Also you can change the default port by `port` field.
